packages/google_maps_flutter/google_maps_flutter_web/example/pubspec.yaml

```diff
@@ -25,3 +25,9 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     sdk: flutter
+  
+# TODO: Remove ovveride before merge
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides: 
+  google_maps_flutter_platform_interface:
+    path: ../../google_maps_flutter_platform_interface
```

Comment:
Why are you overriding this if you aren't changing the web implementation?

Relatedly: is this parameter not supported on web?

Reply:
> Relatedly: is this parameter not supported on web?

No, this feature is not supported on web.

I've just checked the documentation and there's a new `moveCamera` ([in beta](https://developers.google.com/maps/documentation/javascript/reference/3.45/map#Map.moveCamera)) that _"Immediately sets the map's camera to the target camera options, without animation."_, but up until now, all the [`pan*` calls](https://developers.google.com/maps/documentation/javascript/reference/3.45/map#Map.panBy) automatically animated: _"If the change is less than both the width and height of the map, the transition will be smoothly animated."_